crates/dojo-lang/src/plugin_test.rs

```diff
@@ -37,11 +36,7 @@ pub fn test_expand_plugin(
     inputs: &OrderedHashMap<String, String>,
     args: &OrderedHashMap<String, String>,
 ) -> TestRunnerResult {
-    test_expand_plugin_inner(
-        inputs,
-        args,
-        &[Arc::new(BuiltinDojoPlugin), Arc::new(StarkNetPlugin::default())],
```

Comment:
Adding the starknet one was to avoid those errors in the test:
<img width="355" alt="image" src="https://github.com/user-attachments/assets/dcf48a21-0934-4e88-a9ea-2d33fd4d04ed">

But we may just don't  care about them, and keep only dojo expanded as you mentioned. 👍

Reply:
Ok, so it's probably better to have a big file without these errors, so I removed this modification.